response_operations_ui/templates/reset-password.html

```diff
@@ -0,0 +1,90 @@
+{% extends "layouts/base.html" %}
+
+{% set errorType = data['error']['type'] %}
+
+{% block page_title %} Reset Password | Survey Data Collection {% endblock %}
+
+{% block main %}
+
+{% if errorType %}
+
+    {% set errorNotEnteredTitle = "Password is required" %}
+    {% set errorNotEnteredDescription = "Please check the passwords and try again" %}
+
+    {% set errorNoMatchTitle = "Your passwords do not match" %}
+    {% set errorNoMatchDescription = "Please check the passwords and try again" %}
+
+    {% set errorStrongerPasswordTitle = "Your password doesn't meet the requirements" %}
+    {% set errorStrongerPasswordDescription = "Please choose a different password" %}
+
+<div class="panel panel--error">
+    <div class="panel__header">
+        <h1 class="panel__title u-fs-r--b">
+            {{ errorNotEnteredTitle if errorType.password[0] == errorNotEnteredTitle }}
```

Comment:
Is this how frontstage does this?  It's not great, but it works so I'm not gonna suggest changing it >_<